Module4_Labs/Lab7_Zoologist/Cards/111.md

```diff
@@ -4,67 +4,67 @@
 
 <!--concepts{Sorting Algorithms, Dictionaries}-->
 
-First we initiate the function `readFileintoDict()`
+**Step 1**: First we initiate the function `readFileintoDict()`
```

Comment:
11 steps is quite a lot for one card - we should look to combine some steps together